<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>196</red>
        <green>196</green>
        <blue>196</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>196</red>
        <green>196</green>
        <blue>196</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>196</red>
        <green>196</green>
        <blue>196</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>220</red>
        <green>220</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Corbel</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <stylestrategy>PreferAntialias</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>PlaMemo Translator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/images/icon.jpg</normaloff>:/icon/images/icon.jpg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>PlaMemo Translator</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="oldtext">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>201</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI Light</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oldlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OLD:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="newtext">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>201</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI Light</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="newlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NEW:</string>
    </property>
   </widget>
   <widget class="QLabel" name="linelabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Line: 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resetbutton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/reset/images/reset.png</normaloff>:/reset/images/reset.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="applybutton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/apply/images/apply.png</normaloff>:/apply/images/apply.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="leftbutton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/arrowleft/images/arrowleft.png</normaloff>:/arrowleft/images/arrowleft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rightbutton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/arrowright/images/arrowright.png</normaloff>:/arrowright/images/arrowright.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="plamemologo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/logo/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutbutton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/questionmark/images/questionmark.png</normaloff>:/questionmark/images/questionmark.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="transicon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/translation/images/translation.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="translatetext1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>201</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI Light</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Please wait...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="translatetransfer1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/arrowleft/images/arrowleft.png</normaloff>:/arrowleft/images/arrowleft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="translatelogo1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Google Translate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="trashbutton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/trash/images/trash.png</normaloff>:/trash/images/trash.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="translatetext2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>201</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI Light</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Please wait...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="translatetransfer2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/arrowleft/images/arrowleft.png</normaloff>:/arrowleft/images/arrowleft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="translatelogo2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Untitled Translate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="translogoicon1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/google/images/google.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="translogoicon2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/questionmark/images/questionmark.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>21:37:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="chrlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CHR:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="character">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI Light</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="compilebutton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/gear/images/gear.png</normaloff>:/gear/images/gear.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="killapp">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill App</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/close/images/close.png</normaloff>:/close/images/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="chrlabelextra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CHR:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="characterextra">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI Light</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chrnum">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="chrnumextra">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="scenelabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scene: 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sceneleftbutton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/arrowlefts/images/arrowlefts.png</normaloff>:/arrowlefts/images/arrowlefts.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="scenerightbutton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/arrowrights/images/arrowrights.png</normaloff>:/arrowrights/images/arrowrights.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_Console"/>
    <addaction name="separator"/>
    <addaction name="action_CNT"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_CNT">
   <property name="text">
    <string>Character Name Translator</string>
   </property>
  </action>
  <action name="action_Console">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
